src/pages/Hours.jsx

```diff
@@ -0,0 +1,268 @@
+import React, { useState } from 'react';
+import { Table, Input, Button, Typography, Space } from 'antd';
+import { SearchOutlined } from '@ant-design/icons';
+import { AFCBackend } from '../util/utils';
+import './Hours.css';
+
+const { Title } = Typography;
+
+const Hours = () => {
+  const [unapprovedVolunteersData, setUnapprovedVolunteersData] = useState([]);
+  const [selectedHours, setSelectedHours] = useState([]);
+  const [search, setSearch] = useState('');
+
+  const approveHours = async (userId, eventId) => {
+    try {
+      await AFCBackend.get(`/hours/approve/` + userId + '/' + eventId);
+      getUnapprovedVolunteers();
+    } catch (e) {
+      console.log(e.message);
+    }
+  };
+
+  const approveMultipleHours = async selectedHoursList => {
+    try {
+      for (let i = 0; i < selectedHoursList.length; i++) {
+        const userAndEventId = selectedHoursList[i].split(' ');
+        await AFCBackend.get(`/hours/approve/` + userAndEventId[0] + '/' + userAndEventId[1]);
+      }
+      getUnapprovedVolunteers();
+    } catch (e) {
+      console.log(e.message);
+    }
+  };
+
+  const declineHours = async (userId, eventId) => {
+    try {
+      await AFCBackend.get(`/hours/decline/` + userId + '/' + eventId);
+      getUnapprovedVolunteers();
+    } catch (e) {
+      console.log(e.message);
+    }
+  };
+
+  const getUnapprovedVolunteers = async (searchQuery = '') => {
+    try {
+      const { data: unapprovedVolunteers } = await AFCBackend.get(
+        `/hours/unapproved/` + searchQuery,
+      );
+
+      const unapprovedVolunteerHours = [];
+
+      for (let i = 0; i < unapprovedVolunteers.length; i++) {
+        const userId = unapprovedVolunteers[i]['userId'];
+        const { data: userData } = await AFCBackend.get(`/users/` + userId);
+
+        unapprovedVolunteerHours.push({
+          key: unapprovedVolunteers[i]['userId'] + ' ' + unapprovedVolunteers[i]['eventId'],
+          user: userData['firstName'] + ' ' + userData['lastName'],
+          organizations: userData['organization'],
+          event_name: unapprovedVolunteers[i]['event']['name'],
+          date: unapprovedVolunteers[i]['event']['startDatetime'].slice(0, 10),
+          start: unapprovedVolunteers[i]['event']['startDatetime'].slice(11, 16),
```

Comment:
🚫 **[eslint]** <[dot-notation](https://eslint.org/docs/rules/dot-notation)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>["event"] is better written in dot notation.
```suggestion
          start: unapprovedVolunteers[i].event['startDatetime'].slice(11, 16),
```